Changed order of addition for acylation (reaction wasnt working), added 2 protocols, minor other changes

toniyol opened the pull request.

abradle commented:
Great - thanks Tony.

The CI isn't working - because the yaml files in here need updating.
https://github.com/xchem/opentrons/tree/master/examples/errors

Can you take a look - I think it just needs column names updating.

On Wed, Aug 1, 2018 at 1:24 PM toniyol <notifications@github.com> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/xchem/opentrons/pull/14
> Commit Summary
>
>    - change addition row then column or column then row, change in yaml
>    file of order of addition (base later), add a protocol for qc if 1st
>    solvent is not solubilising enough
>    - Merge branch 'master' of https://github.com/toniyol/opentrons
>
> File Changes
>
>    - *M* examples/acylation.yaml
>    <https://github.com/xchem/opentrons/pull/14/files#diff-0> (21)
>    - *M* examples/acylation/Others.csv
>    <https://github.com/xchem/opentrons/pull/14/files#diff-1> (1)
>    - *M* examples/acylation/SM_AcCl.csv
>    <https://github.com/xchem/opentrons/pull/14/files#diff-2> (24)
>    - *M* examples/acylation/SM_Amine.csv
>    <https://github.com/xchem/opentrons/pull/14/files#diff-3> (16)
>    - *M* examples/coupling.yaml
>    <https://github.com/xchem/opentrons/pull/14/files#diff-4> (2)
>    - *M* examples/coupling/Others.csv
>    <https://github.com/xchem/opentrons/pull/14/files#diff-5> (7)
>    - *M* xchem_ot/utils/parser.py
>    <https://github.com/xchem/opentrons/pull/14/files#diff-6> (4)
>    - *M* xchem_ot/utils/protocols.py
>    <https://github.com/xchem/opentrons/pull/14/files#diff-7> (194)
>
> Patch Links:
>
>    - https://github.com/xchem/opentrons/pull/14.patch
>    - https://github.com/xchem/opentrons/pull/14.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xchem/opentrons/pull/14>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFuRIeC1H1qH4HkOS_By7_G3VUBKtwEwks5uMZ34gaJpZM4VqZhb>
> .
>

toniyol commented:
Morning future rich man


See below. I'm trying to understand the process and might be
completely wrong sorry.

Travis says diff with header_error when he compares it to the
header_error yaml file.

The columns (csv headers) of the csv file haven't changed at all so it
shouldn't see a difference. However the yaml file is quite different,
and for example the line: Skipping:
stockSolution_ABtype_troughTo24_single1000. doesnt exist in that yaml
file anymore, it is now:
stockSolution_2reactants_troughTo24_single1000: So it seems like it is
pointing out at things that aren't for this type of header error (if
this error is to spot headings errors on the csv file), but changes of
protocol naming?

Should I change the yaml files in your errors folder to the current
one. Do I make the changes by editing on the github/edit directly?

In terms of yaml file modification, I have changed the naming of some
protocols, added one or two protocols, and added in general_reagents
solvent_QC2: "MeOH", which also means that on the csv file Others
there is a new row

for this solvent.

I am in the lab quite  a lot so might not reply very quickly!

--------------

Difference between header_error
1,6c1,2 < Skipping: stockSolution_ABtype_troughTo24_single1000 <
Skipping: reagentsDispensing_ABtype_24to96_single1000 < Skipping:
reagentsQC&Screen_ABtype_24to96_24to384_troughTo96_single1000 <
Skipping: reactionWUp4mix_troughTo96_multi300_DCM < Skipping:
pWUpBottomPhaseTransfer_96to96_multi300_DCM < Skipping:
pWUpDmsoDispensing_troughTo96_multi300 --- > Column: AC volume p
reaction at 0.8M not found in examples/acylation/SM_AcCl.csv > Options
are: ['AC volume per reaction at 0.8M', 'Barcode rack', 'Barcode
vial', 'Eq', 'Exp weight', 'IUPAC name', 'Location', 'MW', 'Min
weight', 'Name', 'Reaction Scale (mmol)', 'Smiles', 'Volume to
dispense (exp) at 0.8M', 'Volume to dispense (theory) at 0.8M',
'amount per reaction (mg)']

travis_time:end:18523f47:start=1533126353107624173,finish=1533126353853403936,duration=745779763

--------------------

On Wed, 1 Aug 2018 at 13:32, Anthony Bradley <notifications@github.com>
wrote:

> Great - thanks Tony.
>
> The CI isn't working - because the yaml files in here need updating.
> https://github.com/xchem/opentrons/tree/master/examples/errors
>
> Can you take a look - I think it just needs column names updating.
>
> On Wed, Aug 1, 2018 at 1:24 PM toniyol <notifications@github.com> wrote:
>
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/xchem/opentrons/pull/14
> > Commit Summary
> >
> > - change addition row then column or column then row, change in yaml
> > file of order of addition (base later), add a protocol for qc if 1st
> > solvent is not solubilising enough
> > - Merge branch 'master' of https://github.com/toniyol/opentrons
> >
> > File Changes
> >
> > - *M* examples/acylation.yaml
> > <https://github.com/xchem/opentrons/pull/14/files#diff-0> (21)
> > - *M* examples/acylation/Others.csv
> > <https://github.com/xchem/opentrons/pull/14/files#diff-1> (1)
> > - *M* examples/acylation/SM_AcCl.csv
> > <https://github.com/xchem/opentrons/pull/14/files#diff-2> (24)
> > - *M* examples/acylation/SM_Amine.csv
> > <https://github.com/xchem/opentrons/pull/14/files#diff-3> (16)
> > - *M* examples/coupling.yaml
> > <https://github.com/xchem/opentrons/pull/14/files#diff-4> (2)
> > - *M* examples/coupling/Others.csv
> > <https://github.com/xchem/opentrons/pull/14/files#diff-5> (7)
> > - *M* xchem_ot/utils/parser.py
> > <https://github.com/xchem/opentrons/pull/14/files#diff-6> (4)
> > - *M* xchem_ot/utils/protocols.py
> > <https://github.com/xchem/opentrons/pull/14/files#diff-7> (194)
> >
> > Patch Links:
> >
> > - https://github.com/xchem/opentrons/pull/14.patch
> > - https://github.com/xchem/opentrons/pull/14.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/xchem/opentrons/pull/14>, or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AFuRIeC1H1qH4HkOS_By7_G3VUBKtwEwks5uMZ34gaJpZM4VqZhb
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xchem/opentrons/pull/14#issuecomment-409558524>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AYmUHa-YkL-ugTm72F6ZsKWecaMr0STJks5uMZ_DgaJpZM4VqZhb>
> .
>
